sims/common-sim-flags.mk

```diff
@@ -21,3 +21,6 @@ SIM_LDFLAGS = \
 	-lfesvr \
 	-ldramsim \
 	$(EXTRA_SIM_LDFLAGS)
+
+SIM_FILE_REQS += \
+	$(ROCKETCHIP_RSRCS_DIR)/vsrc/EICG_wrapper.v
```

Comment:
Note: If https://github.com/chipsalliance/rocket-chip/pull/2810 goes through then this is unnecessary.